### PR TITLE
[RA1 Ch08] Update chapter08.md

### PR DESCRIPTION
**Pull request by sukhdevkapur**
Fixing the original commit to put it under appropriate header (i.e Section 8.2)
This addresses issue number 372

### CODE REVIEW

File: doc/ref_arch/openstack/chapters/chapter08.md [hunk 1]
```diff
@@ -10,9 +10,27 @@
 
 <a name="8.1"></a>
 ## 8.1 Introduction
+The purpose of this chapter is to identify the gaps between what is required for automated deployment of VNFs on NFVI frameworks and framework offered by OpenStack. 
+Once gaps are identified, the next step will be to propose a plan to address these gaps. 
+The most obvious way to address the gaps will be to propose a set of APIs in the upstream OpenStack community
 
 <a name="8.2"></a>
-## 8.2 Heading
+## 8.2 The Gaps
```
Comment (deebhatia) [inline doc/ref_arch/openstack/chapters/chapter08.md]: Add this in ToC as well.

Comment (rabi-abdel) [reply doc/ref_arch/openstack/chapters/chapter08.md]: fixed it on behalf of @sukhdevkapur

File: doc/ref_arch/openstack/chapters/chapter08.md [hunk 2]
```diff
@@ -5,14 +5,32 @@
...
+
+Service Function Chain - Reference model  makes a reference to SFC support from NFVI. Moreover to support network services, a way is needed to chain the services. Service Function Chain was developed in OpenStack and went dormant. It is not supported and well tested. This requires to be revamped to be useful for CNTT reference architecture
+
+Packet Acceleration Request (e.g Hardware Acceleration) - Reference model makes a reference packet acceleration. OpenStack does not have such support that could be easily used or automated for deployments of VNFs/NFVIs. 
+
+Hardware Interfaces Discovery  - There is no easy way in OpenStack to identify and publish hardware interfaces (NICs and their capabilities). Ironic Inspector has some such capabilities, but it requires a manual intervention.
```
Comment (rabi-abdel) [inline doc/ref_arch/openstack/chapters/chapter08.md]: Can we remove hardware interfaces discovery from the list, hardware interfaces is not something we would like to expose to workloads (VNFs)?

File: doc/ref_arch/openstack/chapters/chapter08.md [hunk 3]
```diff
@@ -5,14 +5,32 @@
...
+
+Support Load Balance of VNF/CNFs - Ability to load balance workflows through multiple instances of same VNF or CNF - e.g. using ECMP to distribute workloads through the multiple instances of Firewall. As an example imagine a distributed finance application with multiple instances of Web-tier and DB-tier. The traffic needs to flow through multiple instances of Firewall (for HA as well load balancing). There is no simple way to accomplish this in OpenStack. 
+
+Service Function Chain - Reference model  makes a reference to SFC support from NFVI. Moreover to support network services, a way is needed to chain the services. Service Function Chain was developed in OpenStack and went dormant. It is not supported and well tested. This requires to be revamped to be useful for CNTT reference architecture
+
+Packet Acceleration Request (e.g Hardware Acceleration) - Reference model makes a reference packet acceleration. OpenStack does not have such support that could be easily used or automated for deployments of VNFs/NFVIs.
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter08.md]: Can we correct "Reference model makes a reference packet acceleration" -- is it a requirement ("must" :should" etc.)

**Comment by rabiabdel** [comment]
@sukhdevkapur can you please address reviews above?

**Comment by rabi-abdel** [comment]
@sukhdevkapur please resolve issues raised above so we can get this merged.

**Comment by pgoyal01** [comment]
@karinesevilla what about the searchlight plugin -- can it not be used to discover resources?

**Comment by sukhdevkapur** [comment]
> @karinesevilla what about the searchlight plugin -- can it not be used to discover resources?

@karinesevilla It is my understanding that searchlight plugin is not suitable for this requirement. It is more like supporting search (from a list - e.g. elastic search)

**Comment by sukhdevkapur** [comment]
@rabi-abdel, @pgoyal01, @karinesevilla - I have addressed your comments. Please have a look.

**Comment by sukhdevkapur** [comment]
Removed idle lable

File: doc/ref_arch/openstack/chapters/chapter08.md (hunk 2 above)
Comment (sukhdevkapur) [reply doc/ref_arch/openstack/chapters/chapter08.md]: I have updated based upon the comment provided by @rabi-abdel

**Comment by pgoyal01** [comment]
Fixes #372 
Issue #372 closed manually

**Pull request merged**